test/cert-ceremonies/root-ceremony-rsa.yaml

```diff
@@ -20,3 +20,14 @@ certificate-profile:
     key-usages:
         - Cert Sign
         - CRL Sign
+skip-lints:
+   - e_ext_authority_key_identifier_missing
+   - e_ext_authority_key_identifier_no_key_identifier
```

Comment:
These two are checking RFC 5280: 4.2.1.1, which doesn't apply to self-signed roots.